src/java.base/share/classes/java/util/zip/ZipFile.java

```diff
@@ -343,6 +343,9 @@ public ZipEntry getEntry(String name) {
      * Closing this ZIP file will, in turn, close all input streams that
      * have been returned by invocations of this method.
      *
+     * @implNote This implementation returns an instance of
+     * {@link java.util.zip.InflaterInputStream}.
```

Comment:
What is the reasoning for this note, do we really want it in the API docs?

Reply:
Hello Alan, this change was done by Volker for the point that I raised in the comment here https://github.com/openjdk/jdk/pull/7986#issuecomment-1081319691

Reply:
> Hello Alan, this change was done by Volker for the point that I raised in the comment here [#7986 (comment)](https://github.com/openjdk/jdk/pull/7986#issuecomment-1081319691)

I think it creates the temptation to cast the returned input stream to InflaterInputStream, it might be better to leave it out.